utils/contracts.py

```diff
@@ -18,6 +18,7 @@ class LogicalOperators(enum.Enum):
     lte = "<="
     eq = "=="
     nt = "!"
+    # neq = "!="
```

Comment:
why commented?

Reply:
I added neq because it was missing from the logical operators. However, there was something wrong with the signature creation for neq so i commented it, instead of removing it again. I don't remember exactly what the problem was but I think instead of creating two Expression Nodes like for the other operators, NebulaStream only creates one Expression Node for "!=" and that was problematic. I wasn't able to fix it for my evaluation so I commented it. We should talk about what to do with it.